Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+
+        // Centered pin
+        if let image =  UIImage(named: "ImageCalloutCentered") {
+            let stretchX = [ImageStretches(first: Float(10), second: Float(15)), ImageStretches(first: Float(45), second: Float(50))]
+            let stretchY = [ImageStretches(first: Float(13), second: Float(16))]
+            let imageContent = ImageContent(left: 10, top: 13, right: 50, bottom: 16)
+
+            try? style.addImage(image.tint(regularTintColor),
+                                id: "imageCentered",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageCentered-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+
+        let stretchX = [ImageStretches(first: Float(16), second: Float(21))]
+        let stretchY = [ImageStretches(first: Float(13), second: Float(16))]
+        let imageContent = ImageContent(left: 16, top: 13, right: 23, bottom: 16)
+
+        // Right-hand pin
+        if let image =  UIImage(named: "ImageCalloutRightHanded") {
+            try? style.addImage(image.tint(regularTintColor),
+                                id: "imageRightHanded",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageRightHanded-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+
+        // Left-hand pin
+        if let image =  UIImage(named: "ImageCalloutLeftHanded") {
+            try? style.addImage(image.tint(regularTintColor),
+                                id: "imageLeftHanded",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageLeftHanded-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+    }
+
+    private func addFeatures() -> FeatureCollection {
```

Comment:
The naming of this method implies side effects, but there aren't any. How about `makeFeatures` instead, following the Swift API naming conventions?

<img width="627" alt="Screen Shot 2021-07-20 at 11 36 01 AM" src="https://user-images.githubusercontent.com/205658/126362093-8fa7459f-992b-49f1-9b91-584f7cc6a69f.png">

https://swift.org/documentation/api-design-guidelines/